receivers/mqtt/mqtt.go

```diff
@@ -102,6 +113,39 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 	return true, nil
 }
 
+func (n *Notifier) buildTLSConfig() (*tls.Config, error) {
```

Comment:
Since there's a new `TLSConfig` added to `receivers` package, why not generalizing this also? I mean, making it a function in the `receivers` package instead that takes `TLSConfig` and returns `tls.Config`.

It would also be similar to how [prometheus](https://github.com/prometheus/common/blob/main/config/http_config.go#L1025) handles it

Reply:
Makes sense! I changed it, but I find the naming a bit confusing 😅 Maybe `ToCryptoTLSConfig` would be better, what do you think?

https://github.com/grafana/alerting/pull/232/commits/1b40be90217cf8c5d635df1d13d24bee1830bfb8

Reply:
Nice, thanks! Yeah, agreed about the naming - it can be a bit confusing. `ToCryptoTLSConfig` sounds better 👍

Reply:
Updated :)